modules/smaatoBidAdapter.js

```diff
@@ -388,11 +426,16 @@ const addOptionalAdpodParameters = (request, videoMediaType, imp) => {
   if (!utils.isEmpty(content)) {
     request.site.content = content
   }
-  imp.video.mimes = videoMediaType.mimes
-  imp.video.startdelay = videoMediaType.startdelay
-  imp.video.linearity = videoMediaType.linearity
-  imp.video.skip = videoMediaType.skip
-  imp.video.protocols = videoMediaType.protocols
-  imp.video.skipmin = videoMediaType.skipmin
-  imp.video.api = videoMediaType.api
+}
+
+function getBidFloor(bidRequest, mediaType, sizes) {
+  if (typeof bidRequest.getFloor === 'function') {
+    const size = sizes.length === 1 ? sizes[0] : '*';
```

Comment:
do we have a test case for `sizes.length != 1` ? cannot see a change for that but maybe it is covered already

Reply:
brilliant :clap: 
added more tests, uncovered bugs, now fixed